app/views/users/index.html.erb

```diff
@@ -1,38 +1,8 @@
-<%= stylesheet_link_tag('index') %>
-<div class="container">
-  <div class="user">
-    <img src="example.com" alt="image">
-    <div>
-      <h1>Username</h1>
-      <p>Number of posts x</p>
-    </div>
-  </div>
-  <div class="user">
-    <img src="example.com" alt="image">
-    <div>
-      <h1>Username</h1>
-      <p>Number of posts x</p>
-    </div>
-  </div>
-  <div class="user">
-    <img src="example.com" alt="image">
-    <div>
-      <h1>Username</h1>
-      <p>Number of posts x</p>
-    </div>
-  </div>
-  <div class="user">
-    <img src="example.com" alt="image">
-    <div>
-      <h1>Username</h1>
-      <p>Number of posts x</p>
-    </div>
-  </div>
-  <div class="user">
-    <img src="example.com" alt="image">
-    <div>
-      <h1>Username</h1>
-      <p>Number of posts x</p>
-    </div>
-  </div>
-</div>
\ No newline at end of file
+<div class="btn">
+  <button>
+    <%= link_to 'Create new post', new_user_post_path(current_user) %>
+  </button>
+</div>
```

Comment:
- [ OPTIONAL ] I suggest removing the "create new post" button from the /users index page and relocating it to the user's show page. This way, you can identify the user you are creating the post for.

Reply:
fixed here [f5a60e4]